vm/src/stdlib/json.rs

```diff
@@ -3,11 +3,15 @@ mod machinery;
 
 #[pymodule]
 mod _json {
-    use crate::function::PyFuncArgs;
+    use super::*;
```

Comment:
We don't have much things in super. maybe `use super::machinery;`?

Reply:
I suppose I could to remove the glob import, but I think it's sorta a convention to have `use super::*` at the top of an inline module, so you don't have to completely "enter" a new scope even though it's the same file.